README.md

```diff
@@ -0,0 +1,96 @@
+[Aspose.Imaging Cloud](https://products.aspose.cloud/imaging/cloud) is a true REST API that enables you to perform a wide range of image processing operations including creation, manipulation and conversion in the cloud, with zero initial costs. Our Cloud SDKs are wrappers around REST API in various programming languages, allowing you to process images in language of your choice quickly and easily, gaining all benefits of strong types and IDE highlights. 
+
+This repository contains Aspose.Imaging Cloud Python SDK source code. This SDK allows you to work with Aspose.Imaging Cloud REST APIs in your Python applications quickly and easily, with zero initial cost.
+
+To use this SDK, you will need App SID and App Key which can be looked up at [Aspose Cloud Dashboard](https://dashboard.aspose.cloud/#/apps) (free registration in Aspose Cloud is required for this).
+
+The solution is updated using [code generator](https://github.com/aspose-imaging-cloud/aspose-imaging-cloud-codegen).
+
+# Features
+### Image Formats Support
+Export the following images to various formats (generally supported ones are BMP, PSD, JPEG, TIFF, GIF, PNG, JPEG2000, WEBP and PDF):
+* BMP
+* GIF
+* DJVU
+* WMF
+* EMF
+* JPEG
+* JPEG2000
+* PSD
+* TIFF
+* WEBP
+* PNG
+* DICOM
+* CDR
+* ODG
+* DNG
+* SVG
+* CMX
+
+Process options, change and return images in the same format:
+* PSD
+* JPEG
+* TIFF
+* GIF
+* BMP
+* JPEG2000
+* WEBP
+
+Process options, change and return images in any supported export format:
+* EMF
+* WMF
+
+### Supported Imaging Operations
+* Export 
+* Resize
+* Crop
+* Rotate and Flip
+* TIFF frames extraction
+* TIFF frames manipulation
+* TIFF concatenation
+* TIFF conversion to fax-friendly format
+* Retrieve & update image properties
+* Conversion to and from PSD format
+
+### Supported Imaging AI Operations
+* Content-based image search
+* Image duplicates search
+* Image search by custom registered tags
+* Image comparison and similarity detection
+* Image features extraction (for now, AKAZE detector is supported)
+
+For the complete list of use-cases, please refer to [common operations format support map](https://docs.aspose.cloud/display/imagingcloud/Supported+File+Formats#SupportedFileFormats-CommonOperationsFormatSupportMap) to see what you can achieve!
+
+# Storage API support
+#### Since version 19.4, SDK includes support of storage operations for better user experience and unification, so now there's no need to use 2 different SDKs!
+
+It gives you an ability to:
+* Upload, download, copy, move and delete files, including versions handling (if you are using Cloud storage that supports this feature - true by default)
+* Create, copy, move and delete folders
+* Copy and move files and folders accross separate storages in scope of a single operation
+* Check if certain file, folder or storage exists
+
+# Usage
+Please, add the following [NuGet package](https://www.nuget.org/packages/Aspose.Imaging-Cloud/) to your project.
```

Comment:
@s-zubov Please, provide the appropriate instructions and future link for package manager instead of NuGet.

Reply:
Thank you, I've added a link to the PyPi website as the package is not published yet.

Reply:
@s-zubov would you mind to add an actual to-be link (as if it was released)? Thanks.

Reply:
@s-zubov Also, please provide an example of how to include this package in the project, considering other SDKs as an example. The reason why it's not done in .NET SDK is that NuGet package page includes this info right away, while other package managers (including PyPI) don't.

Reply:
@s-zubov Please, check your latest commit - there's probably something wrong with _.gitignore_ in this repo or with _.swagger-codegen-ignore_ in Python codegen templates. Only README had to be changed.

Reply:
@s-zubov Seems that I've missed a major issue with package name - it should be **_asposeimagingcloud_** instead of **_aspose-imaging-cloud-sdk_**. Please, change it and update all dependencies (imports, README, links etc). Thanks.

Reply:
@s-zubov sorry for the confusion caused by my recent comment. We had a lot of discussions about naming conventions, and it seems that the right package name is simply _**aspose-imaging-cloud**_. Thanks.

Reply:
@MaxCrank platform-specific guidelines are not taking into consideration? According to PEP, package name should be **asposeimagingcloud**, not **aspose_imaging_cloud** or **aspose-imaging-cloud**. Of course, PEP is just a recommendation.

> Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.
https://www.python.org/dev/peps/pep-0008/#package-and-module-names

Reply:
I've updated package name to **aspose-imaging-cloud** according to company standards.